addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -340,6 +341,9 @@ registerModel({
             compute: '_computeIsDisabled',
             default: false,
         }),
+        isPreparingAttachmentsLoading: attr({
```

Comment:
I think the field is actually useless, it's equivalent to `attachmentsLoaderTimeout !== undefined`?

Reply:
Comment not done, can we look the not introduce this extra field?

Reply:
Done, turned into computed field.

Reply:
Ok, it still looks like a useless indirection, but it's already much better with a compute.